tests/app.test.js

```diff
@@ -634,3 +634,99 @@ describe('InVEST subprocess testing', () => {
     spy.mockRestore();
   });
 });
+describe('Tab closing and switching', () => {
```

Comment:
This test makes sense to me!

As a more general question, `eslint` is showing a bunch of messages on this file.  Are we sticking to the linting rules defined in `.eslintrc`?

Reply:
Yes please! I encourage us to keep adding/removing from the `.eslintrc` rules to override defaults that we don't like, but I do think it's useful to comply with whatever rules we have in place.

Reply:
In those rules, I started using `warn` as opposed to `error` for rules that I sometimes like to follow, but sometimes like to use my own judgment and ignore.

Reply:
Whoops, forgot to lint! Thanks for catching that.